src/logic/proposals/encoding/encoding.ts

```diff
@@ -224,7 +224,7 @@ export const decodeEvmScript = async (
       ['address', 'uint256', 'bytes'],
       utils.hexDataSlice(callData, 4)
     );
-    const targetContractAddress = await tryConvertToEnsName(provider, executionParameters[0]);
+    const targetContractAddress = executionParameters[0];
```

Comment:
I believe this is wrong. The EVM encoding will fail if you try to encode an ENS name (e.g. "bbenligiray.eth") as `address`. So I think we need to convert to ENS here.

Reply:
Also, this encoding is one of the most critical parts of the FE so I find it a bit strange/sad that no tests failed because of this change. Or am I missing something?

Reply:
Hmm this is the `decodeEvmScript` function where the `tryConvertToEnsName` is used just for display purposes afaict. Are you thinking about the encoding function? That has not been touched. 

How I understand it, the encoding process makes sure any potential ENS names are converted to addresses (before creating the proposal), and when decoding, the addresses don't need to be converted into ENS names upfront, because we only convert them to ENS names for display purposes.